src/libs/ComponentUtils/index.native.js

```diff
@@ -1,7 +1,9 @@
 const PASSWORD_AUTOCOMPLETE_TYPE = 'password';
+const NEW_PASSWORD_AUTOCOMPLETE_TYPE = 'password-new';
 const ACCESSIBILITY_ROLE_FORM = 'none';
 
 export {
     PASSWORD_AUTOCOMPLETE_TYPE,
     ACCESSIBILITY_ROLE_FORM,
+    NEW_PASSWORD_AUTOCOMPLETE_TYPE,
```

Comment:
@Harshdeepjoshi can you please rename this file to `index.android.js`?
It'll make it clear that it's android only.